samples/contrib/aws-samples/simple_train_pipeline/README.md

```diff
@@ -41,8 +41,31 @@ An example pipeline with only [train component](https://github.com/kubeflow/pipe
    boto3.resource('s3').Bucket(bucket).Object(data_key).upload_fileobj(buf)
    ```
    Run this file `python s3_sample_data_creator.py`
-3. Prepare an IAM role with permissions to run SageMaker jobs and access to S3 buckets. https://docs.aws.amazon.com/sagemaker/latest/dg/sagemaker-roles.html  
-   Go to you AWS account -> IAM -> Roles -> Create Role -> Select SageMaker Service -> Next -> Next -> Next -> Give "SageMakerExecutorKFP" as Role Name -> Create Role -> Click on the role that you created -> Attach policy -> AmazonS3FullAccess -> Attach Policy -> Note down the role ARN
+3. Prepare an IAM role with permissions to run SageMaker jobs and access to S3 buckets. 
+   ```buildoutcfg
+   # create a new file trust.json with following content 
+   {
+     "Version": "2012-10-17",
+     "Statement": [
+       {
+         "Sid": "",
+         "Effect": "Allow",
+         "Principal": {
+           "Service": "sagemaker.amazonaws.com"
+         },
+         "Action": "sts:AssumeRole"
+       }
+     ]
+   }
+   
+   # run these commands to create a role named "SageMakerExecutorKFP" with SageMaker and S3 access
+   aws iam create-role --role-name SageMakerExecutorKFP --assume-role-policy-document file://trust.json
+   aws iam attach-role-policy --policy-arn arn:aws:iam::aws:policy/AmazonSageMakerFullAccess --role-name SageMakerExecutorKFP
+   aws iam attach-role-policy --policy-arn arn:aws:iam::aws:policy/AmazonS3FullAccess --role-name SageMakerExecutorKFP
```

Comment:
I believe AmazonSageMakerFullAccess actually contains AmazonS3FullAccess within it.

Reply:
No it doesn’t 
I thought so too but it only has some S3 permissions 
But you won't be able to access the data without attaching other S3 permissions